src/main/java/com/blockscore/models/Address.java

```diff
@@ -183,4 +191,6 @@ public String getPostalCode() {
   public String getCountryCode() {
     return countryCode;
   }
+
 }
+
```

Comment:
@archana-na: Hey, sorry for the delay again. I had some other tasks that took precedence over this before the long weekend, but I will be able to review it sometime today for certain.